amical/mf_pipeline/bispect.py

```diff
@@ -989,13 +990,16 @@ def _add_infos_header(infos, hdr, mf, pa, filename, maskname, npix, verbose=True
 
     astropy_version = astropy.__version__
     working_version = "5.0rc1"
-    if Version(astropy_version) < Version(working_version):
+    hdr_commentary_keys = fits.Card._commentary_keywords
+    if any(hck in hdr for hck in hdr_commentary_keys) and (
+        Version(astropy_version) < Version(working_version)
+    ):
         if verbose:
-            cprint(
+            warnings.warn(
                 "Commentary cards are removed from the header with astropy"
                 f" version < {working_version}. Your astropy version is"
                 f" {astropy_version}",
-                "green",
+                RuntimeWarning,
```

Comment:
I researched it (I can never remember what warning types to use), but this seems to be the best choice to me, well done !

Reply:
I also never remember Python warnings, I agree `RuntimeWarning` seems like the best option, but I'm open to suggestions if anyone else has a better idea! I'll mark as resolved for now.